recipes/ebooklib/setup.patch

```diff
@@ -0,0 +1,13 @@
+diff --git ebooklib/setup.py ebooklib/setup.py
+index b3a9a6d..37a25e8 100644
+--- ebooklib/setup.py
++++ ebooklib/setup.py
+@@ -10,7 +10,7 @@
+     url = 'https://github.com/aerkalov/ebooklib',
+     license = 'GNU Affero General Public License',
+     description = 'Ebook library which can handle EPUB2/EPUB3 and Kindle format',
+-    long_description = open('README.md').read(),
++    long_description = 'Windows and py3k had trouble with ingesting README.md here. See README.md directly',
```

Comment:
Turns out this was already fixed in the original repo but not yet merged into a new version for pypi - see [issue 99](https://github.com/aerkalov/ebooklib/issues/99) and [issue 104.](https://github.com/aerkalov/ebooklib/issues/104)